pkgs/by-name/er/errands/package.nix

```
@@ -59,6 +60,10 @@ python3Packages.buildPythonApplication rec {
     makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
   '';
 
+  passthru = {
+    updateScript = gitUpdater { rev-prefix = "v"; };
```

Have you tested this? this does not match the fetchFromGitHub rev

```suggestion
    updateScript = gitUpdater { rev-prefix = ""; };
```